security/csp-report/index.tpml

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>CSP Report leak</title>
+</head>
+<body>
+<p><a href="../index.html">[Home]</a></p>
+
+<p>Leaked CSP report sources:</p>
```

Comment:
Can we add one sentence of explanation? something along the lines ~ "This page will attempt block all injected scripts and leak them through CSP. You will see all scripts leaked this way on the list below"